src/views/SendConfirm.js

```diff
@@ -300,9 +308,7 @@ const SendConfirm = () => {
       </View>
       <Footer>
         <Button
-          disabled={
-            !amount || amount <= 0 || isAmountTooHigh || isPaymentNoteInvalid
```

Comment:
We might want to keep `!amount || amount <= 0`, right?
To be sure that there's an amount to be sent.